packages/PriceLockup/PriceLockup.jsx

```diff
@@ -77,12 +133,9 @@ const PriceLockup = ({ size, price, topText, signDirection, rateText, bottomText
         >
```

Comment:
There are `Box` components above this line that are not styled with Styled Components. This should be possible with the new `Box` update.